dbt_project_create.go

```diff
@@ -0,0 +1,143 @@
+package fivetran
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+)
+
+type DbtProjectCreateService struct {
+	c             *Client
+	groupID       *string
+	dbtVersion    *string
+	gitRemoteUrl  *string
+	gitBranch     *string
+	defaultSchema *string
+	folderPath    *string
+	targetName    *string
+	threads       int
```

Comment:
This should be pointer, otherwise we won't be able to omit the value.

Reply:
Done. :)